.github/workflows/build-mudlet.yml

```diff
@@ -58,11 +66,14 @@ jobs:
         fetch-depth: 0
 
     - name: Install Qt
+      # need to install qt via homebrew for qt5 on arm64-osx
```

Comment:
Could you mention the reason for doing so in the comment please? Just so we remember the 'why'

Reply:
done. let me know if it needs to be more clear.